script/ExecTransaction.s.sol

```diff
@@ -51,26 +47,19 @@ contract ExecTransaction is SafeTxDataBuilder {
     }
 
     function loadSignatures() internal view returns (bytes[] memory signatures) {
-        bytes memory res = bytes(vm.readFile(SIGNATURES_FILE));
-
-        // If the file only contains a single signature, ffi converts it to bytes and can be used as is.
-        if (res.length == 65) {
-            signatures = new bytes[](1);
-            signatures[0] = res;
-        } else {
-            uint256 nbSignatures = res.length / SIGNATURE_LINE_LENGTH;
-            signatures = new bytes[](nbSignatures);
-
-            for (uint256 i; i < nbSignatures; ++i) {
-                uint256 start = 2 + i * SIGNATURE_LINE_LENGTH; // Don't read the first 2 bytes (0x prefix).
-
-                bytes memory signature = new bytes(SIGNATURE_LENGTH);
-                for (uint256 j; j < SIGNATURE_LENGTH; ++j) {
-                    signature[j] = res[start + j];
-                }
-
-                signatures[i] = vm.parseBytes(string(signature));
-            }
+        signatures = new bytes[](THRESHOLD);
+
+        string memory signature;
+        for (uint256 i; i < THRESHOLD; ++i) {
+            signature = vm.readLine(SIGNATURES_FILE);
```

Comment:
shouldn't we skip the first 2 bytes as well?

Reply:
You're right! Tested just now, works with the latest version of the code